app/assets/stylesheets/styles/components/guides/_new_guides.css.scss

```diff
@@ -22,6 +22,148 @@
 }
 
 .new-guide {
+<<<<<<< HEAD
+
+  .guide-creation-nav {
+    $guide-nav-color: #5E9546;
+    background: $guide-nav-color;
+    overflow-x: auto;
+    overflow-y: hidden;
+
+    /* the ul containing the links */
+    ul {
+      list-style: none;
+      margin-left: 0;
+      padding-top: 5px;
+      padding-left: 0;
+      padding-right: 0;
+      text-align: center;
+      white-space: nowrap;
+    }
+
+    li {
+      position: relative;
+      display: inline-block;
+    }
+
+    li a {
+      color: white;
+      font-size: 1.25em;
+      font-weight: 600;
+      vertical-align: middle;
+      color: rgba(255,255,255,0.7);
+      padding: 10px 10px 10px 35px;
+
+      &.disabled {
+        pointer-events: none;
+      }
+
+      &:hover {
+        color: rgba(255,255,255,0.9);
+      }
+
+      &::before {
+        position: absolute;
+        top: 10px;
+        left: 10px;
+        $size: 12px;
+        width: $size; height: $size;
+        content: "";
+        @include box-shadow(
+            0 0 0 2px $guide-nav-color,
+            0 0 0 4px #D9E9D2);
+        border-radius: 50%;
+      }
+
+      &.check::after {
+        @include manual-font-awesome('\f00c');
+        position: absolute;
+        top: 4px;
+        left: 8px;
+        font-size: 22px;
+        color: white;
+      }
+
+      &:active, &.active {
+        color: rgba(255,255,255,1);
+
+        // get rid of check
+        &::after {
+          display: none;
+        }
+      }
+
+      &.active::before {
+        background: #D9E9D2;
+        @include box-shadow(
+            0 0 0 2px $guide-nav-color,
+            0 0 0 4px #D9E9D2);
+      }
+
+      &:hover {
+        // fill in dot
+        &::before {
+          background: white;
+        }
+
+        // get rid of check
+        &::after {
+          display: none;
+        }
+      }
+    }
+
+    // .arrow {
+    //   color: white;
+    //   font-size: 1.45em;
+    //   vertical-align: middle;
+    //   opacity: 0.5;
+    //   margin: auto 10px;
+    // }
+  }
+
+  // TODO: @simonv3, figure out where to put these:
+  .create-guide-steps {
+    text-align: center;
+    font-size: .8rem;
+    color: $of-dark;
+    margin-bottom: 1rem;
+    pointer-events: none;
+
+    .step:first-child {
+      border-top-left-radius: $global-radius;
+      border-bottom-left-radius: $global-radius;
+    }
+
+    .step:nth-child(4){
+      border-top-right-radius: $global-radius;
+      border-bottom-right-radius: $global-radius;
+    }
+
+    .step {
+      padding: .5rem 1rem;
+      border: 1px $secondary-color solid;
+    }
+
+    .step.active {
+      background-color: $of-dark;
+      border: 1px $of-dark solid;
+      color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.